lib/exec.js

```diff
@@ -26,13 +26,19 @@ async function exec (cmd, args = [], opts = {}) {
     logger: undefined,
     maxStdoutBufferSize: MAX_BUFFER_SIZE,
     maxStderrBufferSize: MAX_BUFFER_SIZE,
+    detached: false,
   }, opts);
 
   // this is an async function, so return a promise
   return await new B((resolve, reject) => {
     // spawn the child process with options; we don't currently expose any of
     // the other 'spawn' options through the API
-    let proc = spawn(cmd, args, {cwd: opts.cwd, env: opts.env, shell: opts.shell});
+    let proc = spawn(cmd, args, {
+      cwd: opts.cwd,
+      env: opts.env,
+      shell: opts.shell,
+      detached: opts.detached,
```

Comment:
detached processes should never use STDIO, can it be the case here?

Reply:
Can it be the case that they use stdio? In general? Most likely.

In the particular case I currently need it? No.